rpc/examples/echo/playwright.config.ts

```diff
@@ -0,0 +1,75 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: "./tests",
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 1 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : 3,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: process.env.CI ? "dot" : "html",
```

Comment:
You might need to change this to `html` to have a test report to upload as an artifact; `dot` might not save anything

Reply:
that seems to be the case - however, setting the report to `html` seems cause the step test to hang

Reply:
figured it out, I needed to configure the `html` report to not open: https://github.com/viamrobotics/goutils/pull/221/commits/97d4f6a63301fbeedfd4a153159f755724cb30a7 - curious why we don't have to do this in app tho